lib/python/qmk/cli/doctor/linux.py

```diff
@@ -62,6 +62,9 @@ def check_udev_rules():
             _udev_rule("1eaf", "0003"),  # STM32duino
             _udev_rule("0483", "df11")  # STM32 DFU
         },
+        'wb32': {
```

Comment:
Adding this here without updating the udev rules template is wrong, as it will constantly report issues.